CHANGELOG.md

```diff
@@ -1,32 +1,218 @@
 # Changelog
 
+## Cargo 1.76 (2024-02-08)
+[6790a512...HEAD](https://github.com/rust-lang/cargo/compare/6790a512...HEAD)
+
+### Added
+
+### Changed
+
+### Fixed
+
+- Fixed `--quiet` being used with nested subcommands.
+  [#12959](https://github.com/rust-lang/cargo/pull/12959)
+
+### Nightly only
+
+- 🔥 The `-Zgc` flag enables garbage collection for deleting old, unused files
+  in cargo's cache. That is, downloaded source files and registry index under
+  the `CARGO_HOME` directory.
+  [docs](https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#gc)
+  [#12634](https://github.com/rust-lang/cargo/pull/12634)
+  [#12958](https://github.com/rust-lang/cargo/pull/12958)
+
+### Documentation
+
+### Internal
+
+- Refactored `Cargo.toml` parsing code in preparation of extracting an offical schema API.
+  [#12954](https://github.com/rust-lang/cargo/pull/12954)
+  [#12960](https://github.com/rust-lang/cargo/pull/12960)
+  [#12961](https://github.com/rust-lang/cargo/pull/12961)
+
 ## Cargo 1.75 (2023-12-28)
-[59596f0f...HEAD](https://github.com/rust-lang/cargo/compare/59596f0f...HEAD)
+[59596f0f...rust-1.75.0](https://github.com/rust-lang/cargo/compare/59596f0f...rust-1.75.0)
 
 ### Added
 
+- `package.version` field in `Cargo.toml` is now optional and defaults to `0.0.0`.
+  This field is still required for publishing packages.
+  [#12786](https://github.com/rust-lang/cargo/pull/12786)
```

Comment:
The other thing to note is that `0.0.0` packages are unpublishable.

Reply:
Is it a constraint in crates.io? I don't see anything blocking us from publishing a v0.0.0 package in Cargo.

Reply:
Sorry, I meant packages without a `version` field.

Reply:
Those are blocked within cargo.

Reply:
Tweaked a bit. Thanks for the clarification.